packages/meditrak-server/src/routes/utilities/excel.js

```diff
@@ -32,6 +32,13 @@ export const splitStringOn = (string, splitCharacter) => {
   return string ? string.split(splitCharacter).map(segment => segment.trim()) : [];
 };
 
+export const splitStringOnFirstOccurrence = (string, splitCharacter) => {
+  let [first, ...second] = string ? string.split(splitCharacter) : [];
+  second = second.join(splitCharacter);
+  const result = [first, second];
+  return result.map(segment => segment.trim());
+};
+
```

Comment:
Added this function and also use it in `convertCellToJson` so that `:` can be used in the value as well (I had to use `=` before). 
Eg: value: a:2,b:3,c:4